### PR TITLE
feat: Enable unadorned s-strings in table defs

### PR DESCRIPTION
**Pull request by max-sixty**

### CODE REVIEW

**Comment by aljazerzen** [comment]
> unadorned
[adjective](https://www.merriam-webster.com/dictionary/adjective)
un·​adorned ˌən-ə-ˈdȯrnd 
: not adorned : lacking embellishment or decoration : [plain](https://www.merriam-webster.com/dictionary/plain), [simple](https://www.merriam-webster.com/dictionary/simple)

Damn, I didn't know I needed to know high-english for compiler development :D

File: prql-compiler/src/prql.pest [hunk 1]
```diff
@@ -15,7 +15,7 @@ func_def_param = { ident_part ~ type_def? ~ (":" ~ expr)? }
 type_def = { "<" ~ type_term ~ ( "|" ~ type_term)* ~ ">" }
 type_term = { ident_part ~ type_def? }
 
-table_def = { "table" ~ ident_part ~ "=" ~ nested_pipeline ~ ( NEWLINE+ | &EOI ) }
```
Comment (aljazerzen) [inline prql-compiler/src/prql.pest:18]: It really was that easy. Nice.

**Comment by max-sixty** [comment]
> Damn, I didn't know I needed to know high-english for compiler development :D

rofl, what's a better term? "Raw" / "Un-parenthesized"?

**Review by aljazerzen — APPROVED** [review]

**Comment by aljazerzen** [comment]
> rofl, what's a better term? "Raw" / "Un-parenthesized"?

Bare?

File: prql-compiler/src/prql.pest (hunk 1 above)
Comment (max-sixty) [reply prql-compiler/src/prql.pest:18]: Pest is actually very good at this — would be more difficult in a grammar that restricted ambiguity, like Lezer

**Comment by max-sixty** [comment]
> > rofl, what's a better term? "Raw" / "Un-parenthesized"?
> 
> Bare?

Perfect, #1423

**Pull request merged**